packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -165,6 +165,47 @@ describe('Call Tests', () => {
     expect(call).toBeTruthy();
   });
 
+  it('should log a warning when sending a digit fails', async () => {
+    const tone = '1';
+    const errorMessage = 'Failed to send digit';
+
+    // Mock the mediaConnection object
+    const mockMediaConnection = {
+      insertDTMF: jest.fn(() => {
+        throw new Error(errorMessage);
+      }),
+    };
+
+    const callManager = getCallManager(webex, defaultServiceIndicator);
+
+    const call = callManager.createCall(dest, CallDirection.OUTBOUND, deviceId, mockLineId);
+
+    const realMediaConnection = call['mediaConnection'];
```

Comment:
`call['mediaConnection']` --> call.mediaConnection

Reply:
`const call = callManager.createCall(dest, CallDirection.OUTBOUND, deviceId, mockLineId);` returns an object with interface ICall, which doesn't expose mediaConnection as a member. mediaConnection is a private member of the Call object. Therefore, call.mediaConnection does not work. This is the usual method we've used for all our tests before as well.

Reply:
It makes sense if the type is not exposed but if it is a private member it shouldn't be accessible like this as well unless we treat it as a private member just by not exposing the type.

Reply:
What's a better way of doing this then? Btw I tried doing `call.mediaConnection` and despite typescript throwing an error, the test still passes.

Reply:
> Btw I tried doing call.mediaConnection and despite typescript throwing an error, the test still passes.

This should be because we are not exposing the type but it's in reality not a private member. 

If we want to keep them as "private" and add some checks before setting the value, maybe we can create public getter/setter methods on them.